hydra-cluster/src/Hydra/Cluster/Util.hs

```diff
@@ -141,14 +80,26 @@ chainConfigFor me targetDir nodeSocket them contestationPeriod = do
   forM_ them $ \actor ->
     readConfigFile ("credentials" </> vkName actor) >>= writeFileBS (vkTarget actor)
   pure $
-    defaultChainConfig
-      { nodeSocket
-      , cardanoSigningKey = skTarget me
-      , cardanoVerificationKeys = [vkTarget himOrHer | himOrHer <- them]
-      , contestationPeriod
-      }
+    Direct
+      defaultDirectChainConfig
+        { nodeSocket
+        , hydraScriptsTxId
+        , cardanoSigningKey = skTarget me
+        , cardanoVerificationKeys = [vkTarget himOrHer | himOrHer <- them]
```

Comment:
himOrHer? why not others?

Reply:
I did not change this code, just moved it.